STEERING-COMMITTEE.md

```diff
@@ -161,6 +158,17 @@ There are two ways to raise issues to the steering committee for decision:
    access.
 2. Open an issue on a knative repository and indicate that you would like
    attention from the steering committee.
+   
+   
+## Knative Trademark
+
+**Who owns the Knative Trademark?**
+
+Google owns the Knative word and logo trademarks. Given the open nature of Knative, users may freely use the Knative trademark to accurately refer to the project without prior written permission. Google looks to project leadership to keep an eye out for any misuses of the name. If people are concerned whether their use of the Knative mark is acceptable, we urge them to consult their own counsel or project leadership. 
```

Comment:
'looks to project leadership', what's that? SC, TOC, WG ?

Reply:
SC